packages/common/src/MultiDecimalTestHelper.js

```diff
@@ -0,0 +1,21 @@
+// Run the tests against 2 different kinds of token/synth decimal combinations:
+// 1) matching 18 collateral & 18 synthetic decimals with 18 decimals for price feed.
+// 3) matching 8 collateral & 8 synthetic decimals with 18 decimals for price feed.
+const POSSIBLE_TEST_DECIMAL_COMBOS = [
```

Comment:
nit: Perhaps rename to `TEST_DECIMAL_COMBOS` since `POSSIBLE` seems meaningless. It's used for tests, therefore it must be "possible"

Reply:
sure & done